index.bs

```diff
@@ -1341,7 +1341,8 @@ interface MLGraphBuilder {
   MLOperand input(USVString name, MLOperandDescriptor descriptor);
 
   // Create an operand for a graph constant.
-  MLOperand constant(MLOperandDescriptor descriptor, ArrayBufferView bufferView);
+  MLOperand constant(MLOperandDescriptor descriptor,
+                     [AllowShared] ArrayBufferView bufferView);
```

Comment:
Is there a reason we shouldn't use [`AllowSharedBufferSource`](https://webidl.spec.whatwg.org/#AllowSharedBufferSource)?

Reply:
SGTM. I've updated the algorithm in #787 to take an `AllowSharedBufferSource`. Feel free to update this PR accordingly (I don't think this PR actually has to block on #787, but up to you)

Reply:
Use `AllowSharedBufferSource` in the latest commit. Open a separate comment for #787 dependency. Resolving this one.